android/modules/android/src/java/ti/modules/titanium/android/notificationmanager/NotificationProxy.java

```diff
@@ -384,10 +384,16 @@ public void setNumber(int number)
 	public void setSound(String url)
 	// clang-format on
 	{
+		if (Build.VERSION.SDK_INT >= 26) {
+			Log.w(TAG, "Notification 'sound' property is not supported on Android 8.0 and higher. "
+						   + "You must assign sound to its NotificationChannel instead.");
+		}
```

Comment:
Shouldn't this return?

Reply:
Google's API no-ops on Android 8.0+. So, I didn't see any harm in continuing.

What I'm not sure if I like is logging this every time `Notification.sound` is set. It makes sense to log it to **inform** the app developer that they need to use it for Android 8.0+. But at the same time, they need to set this property for Android 7.1 and below anyways and it's convenient to set this property without an API Level check in the code... making this warning message annoying.